config/alerting.go

```diff
@@ -0,0 +1,55 @@
+// promconfig
+// Copyright (C) 2020 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
I added only percona license, but that code based on configurations from `https://github.com/prometheus/prometheus`. Should I somehow mention that in license?

Reply:
Absolutely yes. We should also use Apache License 2.0.